app/components/draft/draft.test.tsx

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import CompassIcon from '@components/compass_icon';
+import FormattedText from '@components/formatted_text';
+import {General} from '@constants';
+import {renderWithEverything} from '@test/intl-test-helper';
+import TestHelper from '@test/test_helper';
+
+import Draft from './draft';
+
+import type {Database} from '@nozbe/watermelondb';
+import type ChannelModel from '@typings/database/models/servers/channel';
+import type DraftModel from '@typings/database/models/servers/draft';
+
+jest.mock('@components/formatted_text', () => jest.fn(() => null));
+jest.mock('@components/formatted_time', () => jest.fn(() => null));
+jest.mock('@components/compass_icon', () => jest.fn(() => null));
+
+describe('Draft', () => {
+    let database: Database;
+
+    beforeAll(async () => {
+        const server = await TestHelper.setupServerDatabase();
+        database = server.database;
+    });
+    it('should render the draft with channel info and draft message', () => {
+        const props = {
+            channel: {type: General.OPEN_CHANNEL, displayName: 'Direct Message Channel'} as ChannelModel,
+            location: 'channel',
+            draft: {
+                updateAt: 1633024800000,
+                message: 'Hello, World!',
+                channelId: 'channel_id',
+                rootId: '',
+                files: [],
+                metadata: {},
+            } as unknown as DraftModel,
+            layoutWidth: 100,
+            isPostPriorityEnabled: false,
+        };
+        const wrapper = renderWithEverything(
+            <Draft
+                channel={props.channel}
+                location={props.location}
+                draft={props.draft}
+                layoutWidth={props.layoutWidth}
+                isPostPriorityEnabled={props.isPostPriorityEnabled}
+            />
+            , {database},
+        );
+
+        const {getByText} = wrapper;
```

Comment:
nit: can destructure directly or perhaps use wrapper.getByText?
```suggestion
        const {getByText} = renderWithEverything(
            <Draft
                channel={props.channel}
                location={props.location}
                draft={props.draft}
                layoutWidth={props.layoutWidth}
                isPostPriorityEnabled={props.isPostPriorityEnabled}
            />
            , {database},
        );
```